hack/dev/userdata/.master-1.tpl.yaml

```diff
@@ -22,12 +22,28 @@ services:
       apiServer:
         certSANs:
         - 127.0.0.1
+        - 10.5.0.6
         - 10.5.0.7
         - 10.5.0.8
       networking:
         dnsDomain: cluster.local
         podSubnet: 10.244.0.1/16
         serviceSubnet: 10.96.0.1/12
+      etcd:
+        local:
+          serverCertSANs:
+            - master-1
+            - 10.5.0.6
+          peerCertSANs:
+            - master-1
+            - 10.5.0.6
+          extraArgs:
+            initial-cluster: master-1=https://10.5.0.6:2380,master-2=https://10.5.0.7:2380,master-3=https://10.5.0.8:2380
```

Comment:
not for this PR, but looks like this could be auto-generated given that we know all the IPs and names?